README.md

```diff
@@ -2,179 +2,112 @@
-
-- for linux:
-
-```Shell
-cd /usr/lib/node_modules/gtp2ogs/
-sudo npm install
-```
-
-- for windows:
-
-Open a node.js command prompt as admin, then:
+## Options
 
-```Shell
-pushd C:\Users\yourwindowsusername\AppData\Roaming\npm\node_modules\gtp2ogs\
-npm install
-```
+### Options-List
 
-For details or help, you can see:
+You can find the full list of all existing gtp2ogs options [here](/docs/OPTIONS-LIST.md).
 
-- for linux: [3A3) Recommended: Upgrade gtp2ogs.js from old branch to “devel” branch (latest)](https://github.com/wonderingabout/gtp2ogs-tutorial/blob/master/docs/3A3-linux-optional-upgrade-to-devel.md)
-- for windows: [3B3) Recommended: Upgrade gtp2ogs from old branch to devel (latest) branch](https://github.com/wonderingabout/gtp2ogs-tutorial/blob/master/docs/3B3-windows-optional-upgrade-to-devel.md)
+### Options settings
 
-### 4. Most common usage: start gtp2ogs.js using nodejs
+Before putting `options`, you have to first put the mandatory options:
 
-For linux (preferably as sudo):
+- [--username](/docs/OPTIONS-LIST.md#username)
+- [--apikey](/docs/OPTIONS-LIST.md#apikey)
 
-```Shell
-node /usr/lib/node_modules/gtp2ogs/gtp2ogs.js --username <yourbotusername> --apikey <apikey> <gtp2ogsargument1> <gtp2ogsargument2> -- /replace/with/full/path/to/your/ai/runfile.file <botargument1> <botargument2>
-```
+Then, choose the wanted `options` from the above list of all options section
+ (no need to use all these options, only the ones you want), and separate each option with a space.
 
-For windows (preferably as admin):
+note: some options have default so they are enabled even if you don't specify them.
 
-```Shell
-pushd C:\Program Files\nodejs && node.exe C:\Users\yourwindowsusername\AppData\Roaming\npm\node_modules\gtp2ogs\gtp2ogs.js --username <yourbotusername> --apikey <apikey> <gtp2ogsargument1> <gtp2ogsargument2> -- C:\Users\path\to\your\ai\executable.exe <botargument1> <botargument2>
-```
+After that, specify your bot command (and its arguments) using a , add a ` -- `
+ (with one space before and one space after the `--` ), to separate `options`
+ from your bot path and `bot arguments`.
 
-note: for all operating systems, in `--`, the spaces after `<gtp2ogsarguments>`
-and before `/path/to/your/bot.executable` are important: they separate gtp2ogs
-arguments from your bot arguments
+All in all, general use is something like this:
 
-note 2: the number of `<gtp2ogsarguments>` and `<botarguments>` is not limited,
-here only 2 were shown but it possible to use for example 3,4,5 , or as many as
-you want
-  
-note 3: to play on [beta OGS server](https://beta.online-go.com/) instead of the
-[OGS server](https://online-go.com/), add the `--beta` argument
+`path/to/node_executable path/to/gtp2ogs.js --username <yourbotusername> --apikey <apikey> --option1 <argument1> --option2 <argument2> -- path/to/your/ai/executable <botoption1> <botargument1> <botoption2> <botargument2>`
```

Comment:
You should probably make this a code block, it's too long for inline style.

\`\`\`Shell
[code]
\`\`\`

Reply:
This also count for the other blocks below.

Reply:
it was a code block at first, but then i thought it was not clearly visible in one glance

isnt it better to be able to see it all without scrolling?